README.md

```diff
@@ -1,18 +1,46 @@
-A development day project for Suzanne to refresh her Scala knowledge.
 
-# Run Github monitor
+# Pull Request Monitor
+This project started as a development day project for Suzanne to refresh her Scala knowledge.  
```

Comment:
Delete the bit about Suzanne. Reword to: `This project supports the monitoring ...`